packages/liveblocks-node/src/client.ts

```diff
@@ -116,6 +116,18 @@ export type RoomAccesses = Record<
 export type RoomMetadata = Record<string, string | string[]>;
 type QueryRoomMetadata = Record<string, string>;
 
+const RoomTextEditorType = {
+  Lexical: "lexical",
+  TipTap: "tiptap",
+} as const;
+type RoomTextEditorType =
+  (typeof RoomTextEditorType)[keyof typeof RoomTextEditorType];
+
+export type RoomTextEditor = {
+  type: RoomTextEditorType;
```

Comment:
Why not type this directly as `"lexical" | "tiptap"`?

```ts
export type RoomTextEditor = {
  type: "lexical" | "react";
  rootKey: string[];
};
```

Reply:
Really good question here!
TBH I do not know at all, I just followed the same typing than we have in `liveblocks-backend`.  But for sire it can be simplified ✅

Reply:
I think this is copied from mongotypes, the reason it's an enum is in case we support many more editors it will be easier to use.

Reply:
Done in this [commit](https://github.com/liveblocks/liveblocks/pull/2042/commits/bbb3bd8caff0d1fc2a72e86dea4578c2db1a9c8b)